src/api/config/routes.rb

```diff
@@ -345,7 +345,8 @@ def self.public_or_about_path?(request)
     resources :comments, constraints: cons, only: [:create, :destroy], controller: 'webui/comments'
 
     ### /apidocs
-    get 'apidocs(/index)' => 'webui/apidocs#index'
+    get 'apidocs', to: redirect('/apidocs/index')
+    get 'apidocs/(index)' => 'webui/apidocs#index', as: 'apidocs_index'
```

Comment:
Then you probably want this: ``` get 'apidocs((/)index)' => 'webui/apidocs#index' ```

Although it would be nicer to use the expected url schema instead.